app.py

```diff
@@ -14,8 +14,8 @@ def index():
 
 @app.get('/movies')
 def list_all_movies():
-    # TODO: Feature 1
-    return render_template('list_all_movies.html', list_movies_active=True)
+    # Feature 1:  Ashleigh Sico
+    return render_template('list_all_movies.html', list_movies_active=True, movies=movie_repository.get_all_movies())
```

Comment:
Since you changed this line for Ashleigh's feature, make sure her test still work. If they all pass then this should be good.

Reply:
Only one of Ashleigh's test are failing and it isn't because of the code I have written. The only reason I know this is because I ran into the same problem when writing the test for my code. Something about the length of the dictionary not being correct.